server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -161,8 +160,8 @@ export function configureWebSocketServices(
 
             // Validate token signature and claims
             const token = message.token;
-            const claims = jwt.decode(token) as ITokenClaims;
-            if (claims.documentId !== message.id || claims.tenantId !== message.tenantId) {
+            const claims = validateTokenClaims(token, message.id, message.tenantId, isTokenExpiryEnabled);
+            if (!claims) {
                 return Promise.reject("Invalid claims");
```

Comment:
"Invalid access token" to be consistent.

Reply:
Keep "invalid claims" for now. We need to look at the driver